client/app/src/Entity/User.php

```diff
@@ -78,6 +78,12 @@ class User implements UserInterface, DeputyInterface, PasswordAuthenticatedUserI
     public const ACTIVATE_TOKEN_EXPIRE_HOURS = 48;
     public const PASSWORD_TOKEN_EXPIRE_HOURS = 1;
 
+    public const SELF_REGISTER = 'SELF_REGISTER';
+    public const ADMIN_INVITE = 'ADMIN_INVITE';
+    public const ORG_ADMIN_INVITE = 'ORG_ADMIN_INVITE';
+    public const CO_DEPUTY_INVITE = 'CO_DEPUTY_INVITE';
+    public const UNKNOWN_REGISTRATION_ROUTE = 'UNKNOWN';
```

Comment:
Are we going to populate all fields in the table? 
If not then why do we have an unknown value?
If we do plan to populate this field for all users, where is the migration script to update these?

Reply:
Have just seen in the migrations file, that unknown will be the default. So this is required.